Directory.Build.props

```diff
@@ -1,18 +1,33 @@
 <Project>
+    <!-- Make F# support Central Package Management -->
     <PropertyGroup>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+        <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    </PropertyGroup>
+    <!-- Use lock files -->
+    <PropertyGroup Condition="'$(GITHUB_ACTIONS)' == 'true'">
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+    </PropertyGroup>
+    <PropertyGroup>
+        <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+        <RestoreLockedMode Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</RestoreLockedMode>
+    </PropertyGroup>
+    <PropertyGroup>
+        <PackageProjectUrl>https://github.com/ionide/KeepAChangelog</PackageProjectUrl>
```

Comment:
You shouldn't need to add this or the `RepositoryUrl` Property below because SourceLink in the .NET SDK should provide them automatically. If that's not happening, then we should fix that instead of providing duplicate information.